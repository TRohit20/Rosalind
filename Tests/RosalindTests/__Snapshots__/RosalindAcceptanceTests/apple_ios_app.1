{
  "artifactType" : "app",
  "children" : [
    {
      "artifactType" : "file",
      "path" : "App",
      "shasum" : "78c029c60a7fc96d103409024a31a65f64db44eedce46436bec6e307e6d74b76",
      "size" : 70880
    },
    {
      "artifactType" : "file",
      "path" : "App.debug.dylib",
      "shasum" : "402cd1e7641cd8bf46d39ae60084247f369f25ab922a7914d65d2b19039f8ead",
      "size" : 126488
    },
    {
      "artifactType" : "file",
      "path" : "Info.plist",
      "shasum" : "d27f213cc0f6fecc1f3711ae343b264a332b4b087531bb77f81fbae2ec7e0933",
      "size" : 1241
    },
    {
      "artifactType" : "file",
      "path" : "PkgInfo",
      "shasum" : "82502191c9484b04d685374f9879a0066069c49b8acae7a04b01d38d07e8eca0",
      "size" : 8
    },
    {
      "artifactType" : "file",
      "path" : "__preview.dylib",
      "shasum" : "537e6fa5ef09aafaa90e929cdc259b90277444f09fc645a9429a7b8bfba377cb",
      "size" : 16464
    }
  ],
  "path" : ".",
  "shasum" : "9cce125ec6f8f2d69280a7192dfc8ead4f2496dacc8faf4d5e096171b96401aa",
  "size" : 215081
}