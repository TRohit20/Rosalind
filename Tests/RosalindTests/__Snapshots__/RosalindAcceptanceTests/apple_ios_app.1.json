{
  "app" : {
    "children" : [
      {
        "unknown" : {
          "children" : [

          ],
          "path" : "App"
        }
      },
      {
        "unknown" : {
          "children" : [

          ],
          "path" : "App.debug.dylib"
        }
      },
      {
        "unknown" : {
          "children" : [

          ],
          "path" : "Info.plist"
        }
      },
      {
        "unknown" : {
          "children" : [

          ],
          "path" : "PkgInfo"
        }
      },
      {
        "unknown" : {
          "children" : [

          ],
          "path" : "__preview.dylib"
        }
      }
    ],
    "path" : "."
  }
}